File: Programming-Basics-with-Python/1.First-Steps-in-Coding-Lab/09_yard_greening.py
square_meters_of_landscaping = float(input())
sum = square_meters_of_landscaping * 7.61
discount = sum * 0.18
final_price_of_the_service = sum - discount
print(f"The final price is: {final_price_of_the_service} lv.")
print(f"The discount is: {discount} lv.")

File: Programming-Basics-with-JavaScript/1.First-Steps-in-Coding/09yardGreening.js
function yardGreening(input){
    let sqMeters = Number(input[0])
    let discount = (sqMeters * 7.61 ) * 0.18
    let totalSum = sqMeters * 7.61 - discount
    console.log(`The final price is: ${totalSum} lv.`)
    console.log(`The discount is: ${discount} lv.`)
}
yardGreening(["550"])